core/src/main/java/org/apache/druid/java/util/common/parsers/DelimitedParser.java

```diff
@@ -74,6 +77,24 @@ public DelimitedParser(
   @Override
   protected List<String> parseLine(String input)
   {
-    return splitter.splitToList(input);
+    return splitToList(input);
+  }
+
+  /**
+   * Copied from Guava's {@link Splitter#splitToList(CharSequence)}.
```

Comment:
Do we need to update NOTICE? (I'm not sure)

Reply:
After reading http://www.apache.org/dev/licensing-howto.html's "BUNDLING AN APACHE-2.0-LICENSED DEPENDENCY" section I think we don't need to update NOTICE.

Reply:
Thanks.